### PR TITLE
Add publication references to genbank parser

### PR DESCRIPTION
**Pull request by mkleen**

### CODE REVIEW

File: biojava-core/src/test/resources/NM_000266.gb [hunk 1]
```diff
@@ -120,7 +120,7 @@ FEATURES             Location/Qualifiers
                      /gene="NDP"
                      /gene_synonym="EVR2; FEVR; ND"
                      /note="Norrie disease (pseudoglioma)"
-                     /db_xref="GeneID:4693"
+                     /db_xref="         :4693"
                      /db_xref="HGNC:7678"
```
Comment (andreasprlic) [inline biojava-core/src/test/resources/NM_000266.gb]: Hi,

not sure why you are removing the text GeneID here? this is content that is also in the original file:

https://www.ncbi.nlm.nih.gov/nuccore/223972707?report=genbank

**Comment by mkleen** [comment]
I will fix the testcase, i have also a few more commits.

File: biojava-core/src/main/java/org/biojava/nbio/core/sequence/features/PublicationReference.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
+package org.biojava.nbio.core.sequence.features;
+
+import java.util.ArrayList;
+import java.util.List;
+
```
Comment (lafita) [inline biojava-core/src/main/java/org/biojava/nbio/core/sequence/features/PublicationReference.java]: Could you document this Class with javadoc?

File: biojava-core/src/main/java/org/biojava/nbio/core/sequence/features/PublicationReferenceAuthor.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
+package org.biojava.nbio.core.sequence.features;
+
```
Comment (lafita) [inline biojava-core/src/main/java/org/biojava/nbio/core/sequence/features/PublicationReferenceAuthor.java]: Could you document this Class with javadoc?

File: biojava-core/src/test/java/org/biojava/nbio/core/sequence/io/GenbankCookbookTest.java [hunk 4]
```diff
@@ -64,6 +64,7 @@ public void tearDown() {
 	/**
 	 * Test of process method, of class GenbankReader.
 	 */
+	@Ignore
```
Comment (lafita) [inline biojava-core/src/test/java/org/biojava/nbio/core/sequence/io/GenbankCookbookTest.java]: Why do we ignore this test now?

**Comment by lafita** [comment]
Dear @mkleen, what is the status of this PR?

Could you also provide a bit more description of what is added (new features) in this PR?

**Comment by lafita** [comment]
This has been open for more than 6 months and it already has conflicts with the master branch. I would suggest closing it with the possibility of reopening it in the future.

**Comment by josemduarte** [comment]
Yes, let's close. There was no response to the comments for a long time.

**Pull request closed without merging**